### DISCUSSION TITLE
Histograms x-axis scaling issue

### DISCUSSION THREAD

**Discussion by gganapat** [WLJSTeam/wljs-notebook#171] [General]
Hi:

I've been playing around with some examples, and I tried Histogram which is supposed to render like this:

<img width="492" alt="Screenshot 2024-05-20 at 5 12 15 PM" src="https://github.com/JerryI/wolfram-js-frontend/assets/46061218/d4ccae9e-3c39-4b43-ae19-8519b53882f5">

but instead renders like this:

<img width="566" alt="Screenshot 2024-05-20 at 5 12 25 PM" src="https://github.com/JerryI/wolfram-js-frontend/assets/46061218/3ee389cf-c30c-4f33-9f3b-8619d845c189">

Trying to change the AxesOrigin->{-5,0} (or any other value) does nothing meaningful.

Thanks!
Gani --

**Comment by JerryI** [comment]
Hi Gani,
I have sent your post to github issues. it is a problem of autoscale, `Histogram` does not provide `PlotRange` explicitly and `Graphics` container tries to guess the correct one based on data.

It is possible to fix this. As a temporal solution- try to add `PlotRange` option to `Histogram`.

**Comment by JerryI** [comment]
Fixed in #172 

PS: As usual, you need to force updates check from the settings menu. Restart an app and reset browser's cache :)

**Reply by gganapat** [reply]
Thanks!
Gani-
